src/utilities/formatToProposal.ts

```diff
@@ -81,7 +81,14 @@ const formatToProposal = ({
     endDate = now;
   }
 
-  let descriptionObj = { version: 'v1' as const, title: '', description: '' };
+  let descriptionObj: Proposal['description'] = {
```

Comment:
This shouldn't change. v1 descriptions are not changed by the v2 endpoint. v1 descriptions are markdown where as v2 are JSON